docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -17,3 +17,72 @@ on the default backend.
         commands:
             - python "${helloworld}"
 ```
+
+## Custom memory limit
+
+When jobs exceeds the memory limits, you will get out of memory (OOM) error and job gets [`OOMKilled`](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits). With REANA, you can configure cluster wide **memory** limits to make the efficient use of available resources. Note that you can also decrease the memory limit to increase the chances of jobs being scheduled earlier.
```

Comment:
```suggestion
When jobs exceed the memory limits, you will get out of memory (OOM) error and job gets [`OOMKilled`](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits). With REANA, you can configure cluster wide **memory** limits to make the efficient use of available resources. Note that you can also decrease the memory limit to increase the chances of jobs being scheduled earlier.
```

Reply:
> With REANA, you can configure cluster wide **memory** limits to make the efficient use of available resources.

This is not relevant for the end-user, only for admins.

Here we should mention something like..

```
In order to avoid this error,  you can increase the **memory** limits per workflow step to make an efficient use of available resources. 
```